mppb/util.go

```diff
@@ -0,0 +1,117 @@
+package main
```

Comment:
Você vai precisar de tudo isso? Recomendo você remover o arquivo do PR e só colocar o arquivo quando for utilizá-lo.